net/chrony/files/chrony.config

```diff
@@ -1,19 +1,35 @@
 config pool
 	option hostname '2.openwrt.pool.ntp.org'
 	option maxpoll '12'
-	option iburst 'yes'
+	option maxsources '4'
+	option iburst '1'
+	option prefer '1'
+	# option xleave '0'
 
-config dhcp_ntp_server
-	option iburst 'yes'
-	option disabled 'no'
+config dhcp_ntp_server 'dhcp_ntp_server'
+	option iburst '1'
+	option disabled '0'
 
-config allow
+config allow 'allow'
 	option interface 'lan'
 
-config makestep
+config makestep 'makestep'
 	option threshold '1.0'
 	option limit '3'
 
-config nts
-	option rtccheck 'yes'
-	option systemcerts 'yes'
+config nts 'nts'
+	option rtccheck '1'
+	option systemcerts '1'
+
+#config smoothtime 'smoothtime'
+	# option maxppm '400'
+	# option maxwander '0.01'
+	# option leaponly '0'
+
+#config systemclock 'systemclock'
+	# option precision '8e-6'
+	# option leapsecmode 'slew'
+	# option leapsectz 'right/UTC'
+
+#config logging 'logging'
+	#option logchange '0.01'
```

Comment:
Just for consistency, there is no space between `#` and `option` here, but there is a space in the sections above that.